content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  Richard Belleville (Google),
+  Frank Budinsky (IBM),
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Mattia Lavacca (Kong),
+  Christine Kim (Isovalent),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other reviewer and release note contributors.
+---
+
+![Gateway API Logo](gateway-api-logo.svg)
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release of
+[Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentReference Port**,
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. Clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to _Standard_ of four eagerly awaited features.
+This means they are no longer experimental concepts. Inclusion in the Standard
+release channel denotes a high
+level of confidence in the API surface and provides guarantees of backward
+compatibility.  Though these features are now considered stable, that does not
+mean that they are complete. These and other features will continue to evolve via
+the Experimental channel. For more information on how all of this works, refer
+to the [Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+
+#### [GAMMA](https://gateway-api.sigs.k8s.io/mesh/)
```

Comment:
```suggestion
#### [Service Mesh Support](https://gateway-api.sigs.k8s.io/mesh/)
```

Reply:
This may just be my own bias showing through, but I think we've generally been moving away from GAMMA terminology in favor of "Service Mesh support". Ultimately will defer to preference of @howardjohn, @keithmattix, @kflynn, or @mikemorris on this one though.

Reply:
Strong +1

Reply:
Yeah, I agree. We use GAMMA as synecdoche a lot, but "Service Mesh Support" is better for public consumption.

Reply:
+1, it's been easier to understand for new audiences when I've been explaining this as "Gateway API for service mesh"

Reply:
+1 on service mesh instead of GAMMA

Reply:
Done.